src/components/ScholarshipCard.jsx

```diff
@@ -107,6 +110,7 @@ export default function ScholarshipCard({ scholarship, style }) {
   }, [author, currentUser, preview]);
 
   const CardAreaComponent = detailed ? Box : CardActionArea;
+  const lintIssues = canEdit ? lint(scholarship.data) : [];
```

Comment:
It would also be nice to see the warnings when adding a scholarship.

Reply:
oh, I hadn't noticed it didn't appear for new scholarships. Fixed.